code/modules/mob/stats_skills.dm

```diff
@@ -58,11 +58,10 @@
 
 /mob/proc/learn_stats(var/stat_type)
 	var/initial_stat = round(stats[stat_type])
-	if(stat_to_modifier(stats["int"]) > 0)
-		stats[stat_type] += 0.01 * stat_to_modifier(stats["int"]) //This has to be 
+	if(stats[stat_type] > 35)
+		return 0
 	else 
-		if(stats[stat_type] < 35)
-			return 0
+		stats[stat_type] += 0.01 * stat_to_modifier(stats["int"]) //This has to be 
```

Comment:
We still need to check for "stat_to_modifier(stats["int"]) > 0" or else people with less then 10 INT will never learn.

Reply:
this doesnt matter here

Reply:
It will, if they have 8 or less INT it will actually start lowering their stats